mltsp/predict.py

```diff
@@ -46,7 +45,10 @@ def model_predictions(featureset, model, return_probs=True):
     if preds_df.shape[1] == 1:
         preds_df.columns = ['prediction']
     else:
-        preds_df.columns = model.classes_
+        if "GridSearchCV" in str(type(model)):
```

Comment:
This doesn't seem like the best way to check...would `isinstance(model, GridSearchCV)` not work? Are there other possible options? Maybe check inheritance from some common base class?